projects/website/src/plots/PlotDriver.py

```diff
@@ -0,0 +1,323 @@
+                                continue 
+                            elif key_1.lower() == "Y1Label".lower():
+                                job.Y1Label = value_1
+                                continue 
+                            elif key_1.lower() == "X2Label".lower():
+                                job.X2Label = value_1
+                                continue 
+                            elif key_1.lower() == "Y2Label".lower():
+                                job.Y2Label = value_1
+                                continue 
+                            elif key_1.lower() == "ExperimentalData".lower():
+                                job.experimentalData = value_1
+                                continue 
+                            elif key_1.lower() == "DataFileOverride".lower():
+                                job.dataFile = value_1
+                                continue 
+                            elif key_1.lower() == "DataPathOverride".lower():
+                                job.dataPath = value_1
+                                continue 
+                            elif key_1.lower() == "PFTFile".lower():
+                                job.PFTFile = value_1
+                                continue 
+                            elif key_1.lower() == "OutputOverride".lower():
+                                job.outputDir = value_1
+                                continue 
+                            elif key_1.lower() == "FontSize".lower():
+                                job.fontSize = int(value_1)
+                                continue 
+                            else:
+                                logging.warning("Unrecognized config directive: " + directive)   
+            fin.close()
+        except Exception as e:
+            logging.error("Ouch Something went wrong")
+
+    def execute(self):
+        for job in self.jobs:
+            if not job.ignore:
+                try:
+                    if self.jobname==job.name:
+                        job.logger=False
+                    else:
+                        job.logger=True
+                    if job.plotname=="ActionEvent":
+                        job.plotter.plot(job)
+                    self.jobname=job.name
+                except Exception as e:
+                    logging.error("Plotter couldn't plot job " + job.name + ". Check your config file line.")
+                    continue 
+        job.Reset()
+
+if __name__ == '__main__':
+    import sys
+    parser = argparse.ArgumentParser(description='Creation of Plots')
+    parser.add_argument('-p','--configpath',help='Path to Config file',required=True)
+    parser.add_argument('-b','--baseline',help='Baseline Dir path',required=True)
+    args=parser.parse_args()
+    if args.configpath==None and args.baseline==None:
+        parser.print_help()
+        sys.exit(0)
+    else:
+        argv=["PlotDriver.py",args.baseline,args.configpath]
```

Comment:
The first argument is typically the scripts name.  Your assuming a lot by not referring to it by sys.argv[0]

Reply:
Ok, I will change it :+1: